src/utils/reporter.ts

```diff
@@ -41,11 +46,15 @@ export function reportSuccessFunctions(functions: DeployCodeFunctionResponse[])
 
     functions.forEach((func) => {
         functionDeploymentsString += `  - ${func.name}: ${colors.yellow(func.cloudUrl)}` + "\n";
+        if (func.authToken) {
+            functionDeploymentsString +=
+                `  - ${func.name} authentication token: ${colors.green(func.authToken)}` + "\n";
```

Comment:
I am not a fan of revealing this information to the command line without a specific request from the user. Because it's sensitive info - if I am deploying in a public repository in GitHub Actions, this deployment secret id will be leaked.

I advise to either add a new command  something like `genezio reveal <function_cloud_id>` that would print secret informatio or to make it a frontend feature where it should have a reveal button.

Reply:
Also, user facing I would call this token - `deployment secret id`, it seems to me it's more descriptive.